unity/EditorPlugin/UnitTesting/UnityEditorTestLauncher.cs

```diff
@@ -58,9 +64,10 @@ public void TryLaunchUnitTests()
         string testEditorAssemblyProperties =  testEditorAssembly.GetType().GetProperties().Select(a=>a.Name).Aggregate((a, b)=>a+ ", "+b);
         string testEngineAssemblyProperties = testEngineAssembly.GetType().GetProperties().Select(a=>a.Name).Aggregate((a, b)=>a+ ", "+b);
 
```

Comment:
I think this just gets a list of the properties on `Assembly` - is there a missing type name in the call to `GetType()`?